src/Orchard/Security/Providers/FormsAuthenticationService.cs

```diff
@@ -10,7 +12,7 @@
 
 namespace Orchard.Security.Providers {
     public class FormsAuthenticationService : IAuthenticationService {
-        private const int _cookieVersion = 3;
+        private const int _cookieVersion = 4;

```

Comment:
Maybe the `IUserDataValidationProvider`s need to contribute to the cookie version number somehow?